src/mds/MDCache.h

```diff
@@ -449,9 +449,10 @@ class MDCache {
 				   int linkunlink, SnapRealm *prealm);
   void _project_rstat_inode_to_frag(CInode::mempool_inode & inode, snapid_t ofirst, snapid_t last,
 				    CDir *parent, int linkunlink, bool update_inode);
-  void project_rstat_frag_to_inode(nest_info_t& rstat, nest_info_t& accounted_rstat,
+  void project_rstat_to_inode(nest_info_t& rstat, nest_info_t& accounted_rstat,
```

Comment:
don't do unnecessary rename

Reply:
@ukernel Hi:-)

I thought "project_rstat_frag_to_inode" should take "CDir" and "CInode" as parameters like "project_rstat_inode_to_frag", but the current code does not. It seems that this is because we need to project "dirty_old_rstat" into inode in finish_scatter_gather_update. So, I changed the method's name to project_rstat_to_inode and added a new project_rstat_frag_to_inode method whose parameter list is similar to "project_rstat_inode_to_frag". project_rstat_to_inode is called by both "project_rstat_frag_to_inode" and "finish_scatter_gather_update".

Is this more appropriate?